src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,43 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  padding: 0 50px;
+  margin-bottom: 50px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  overflow: hidden;
+
+  &__logo {
+    height: 40px;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    position: relative;
+    color: #060b35;
+    list-style: none;
+    line-height: 60px;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    color: inherit;
+    text-decoration: none;
+    text-transform: uppercase;
```

Comment:
and text-decoration send to noralize maybe too

Reply:
thanks. I was completely forgot about the header